src/format.rs

```diff
@@ -100,3 +127,242 @@ impl fmt::Display for BinaryBytes {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    const NANO: Duration = Duration::from_nanos(1);
+    const MILLI: Duration = Duration::from_millis(1);
+    const SECOND: Duration = Duration::from_secs(1);
+    const MINUTE: Duration = Duration::from_secs(60);
+    const HOUR: Duration = Duration::from_secs(60 * 60);
+    const DAY: Duration = Duration::from_secs(24 * 60 * 60);
+    const WEEK: Duration = Duration::from_secs(7 * 24 * 60 * 60);
+    const YEAR: Duration = Duration::from_secs(365 * 24 * 60 * 60);
+
+    #[test]
+    fn human_duration_less_than_one_second() {
```

Comment:
Let's test `Debug` and `Display` once per function and pick one to run all the other tests. Then, we can also inline the value definition, which makes everything a lot more compact.